### ISSUE TITLE
Default values of Blender nodes

### ISSUE THREAD

**Issue by wouldntsavezion** [KhronosGroup/glTF-Blender-Exporter#250]
Hi, I've only been using the glTF exporter (And the gltf/glb format itself) for 2 days but there's something I noticed with the exporter and I'm not sure if this is intended behavior or this has been discussed before (Quick search gave no results) :

The exporter doesn't consider the values of the node itself for what seems to be a few things, in my case especially the "BaseColor". As I can see in [gltf2_generate.py](https://github.com/KhronosGroup/glTF-Blender-Exporter/blob/19ba449d98376a0e4dbd6406ea0219c94ee89e27/scripts/addons/io_scene_gltf2/gltf2_generate.py#L2182) (on line 2182 for the cycles part), it only checks if there's an accessible texture index and otherwise just proceeds with the generation. As I said there might be a good reason for this but it is possible to have a more "expected" behavior by just adding an else that sets `pbrMetallicRoughness['baseColorFactor']` to `blender_node.inputs[0].default_value`

tldr; This will not result in a red material :
![image](https://user-images.githubusercontent.com/17103048/41670278-d9269b7a-7481-11e8-90bf-1fbcac10b67f.png)
